src/kernels/ipywidgets-message-coordination/ipyWidgetMessageDispatcher.ts

```diff
@@ -106,7 +106,7 @@ export class IPyWidgetMessageDispatcher implements IIPyWidgetMessageDispatcher {
     public receiveMessage(message: IPyWidgetMessage): void {
         switch (message.message) {
             case IPyWidgetMessages.IPyWidgets_logMessage:
-                traceInfoIfCI(`Widget Message: ${message.payload}`);
+                traceInfo(`Widget Message: ${message.payload}`);
```

Comment:
Made this traceInfo because after these two changes, this is really only stuff that we actually want to show to the user (or us when request logs).